core/meterer/meterer.go

```diff
@@ -371,8 +371,8 @@ func ValidateReservations(
 	reservations map[core.QuorumID]*core.ReservedPayment,
 	quorumConfigs map[core.QuorumID]*core.PaymentQuorumProtocolConfig,
 	quorumNumbers []uint8,
-	timestamp int64,
-	receivedAt time.Time,
+	paymentHeaderTimestampNs int64,
```

Comment:
We could remove the above parameters section in the comment, doesn't seem that useful now that we mark Ns in the name